src/Take.Blip.Builder.UnitTests/Actions/ProcessHttpActionTests.cs

```diff
@@ -309,5 +313,49 @@ public async Task ProcessAction_CheckConfigurationVariableValues(string botIdent
                 Arg.Is<HttpRequestMessage>(
                     h => h.RequestUri.Equals(settings.Uri)), Arg.Any<CancellationToken>());
         }
+
+        [Fact]
+        public async Task ProcessAction_AddBotIdentifierAndStateId()
+        {
+
+            var settings = new ProcessHttpSettings
+            {
+                Uri = new Uri("https://blip.ai"),
+                Method = HttpMethod.Post.ToString(),
+                Body = "{\"plan\":\"Premium\",\"details\":{\"address\": \"Rua X\"}}",
+                Headers = new Dictionary<string, string>()
+                {
+                    {"Content-Type", "application/json"},
+                    {"Authorization", "Key askçjdhaklsdghasklgdasd="}
+                },
+                ResponseBodyVariable = "httpResultBody",
+                ResponseStatusVariable = "httpResultStatus",
+
+            };
+
+            var target = GetTarget();
+
+            var httpResponseMessage = new HttpResponseMessage()
+            {
+                StatusCode = HttpStatusCode.Accepted,
+                Content = new StringContent("Some result")
+            };
+
+            HttpRequestMessage requestMessage = null;
+            HttpClient
+                .SendAsync(Arg.Do<HttpRequestMessage>(m => requestMessage = m), Arg.Any<CancellationToken>())
+                .ReturnsForAnyArgs(httpResponseMessage);
+
+            // Act
+            await target.ExecuteAsync(Context, JObject.FromObject(settings), CancellationToken);
+
+            // Assert
+            requestMessage.Headers.Contains("X-Blip-Bot").ShouldBeTrue();
+            requestMessage.Headers.Contains("X-Blip-StateId").ShouldBeTrue();
```

Comment:
this souldn't be true when URI not matches with internal URI...
is this test checked with success?

Reply:
Done